src/server/generic_family.cc

```diff
@@ -1037,8 +1037,15 @@ struct SortEntry
 
   bool Parse(std::string&& item) {
     if constexpr (!ALPHA) {
-      if (!absl::SimpleAtod(item, &this->score))
+      if (!absl::SimpleAtod(item, &this->score)) {
```

Comment:
is "nan" parsed by this or not? If it does, then `this->score == nan` right ?

Reply:
yes it parsed nan and this->score == nan